database/migrations/2018_08_15_103716_move_photos.php

```diff
@@ -35,7 +35,9 @@ public function up()
 				if ($result->album === 0) {
 					$photoAttributes['album_id'] = null;
 				} else {
-					$photoAttributes['album_id'] = Helpers::trancateIf32($result->album, 0);
+					$albumID = Helpers::trancateIf32($result->album, 0);
+					$exists = DB::table('albums')->select('*')->where('id', '=', $albumID)->count() !== 0;
```

Comment:
My gut tells me that
```suggestion
					$exists = DB::table('albums')->select('id')->where('id', '=', $albumID)->count() > 0;
```
would be better, but any difference would be negligible and is probably optimised away anyway.